### PR TITLE
add ImportCompactor and ShadowDistanceChanger

### PR DESCRIPTION
**Pull request by art0007i**

### CODE REVIEW

**Comment by Nytra** [comment]
ImportCompactor repo is private?

**Comment by art0007i** [comment]
yeah it is. ooops, fixed it just now

I use visual studio to create git repos for my mods and it defaults to private and sometimes I forget to untick it...

File: manifest.json [hunk 1]
```diff
@@ -4830,6 +4830,29 @@
                 }
             }
         },
+        "me.art0007i.ImportCompactor": {
```
Comment (XDelta) [inline manifest.json:4833]: +1

File: manifest.json [hunk 2]
```diff
@@ -4997,6 +5020,29 @@
                 }
             }
         },
+        "me.art0007i.ShadowDistanceChanger": {
```
Comment (XDelta) [inline manifest.json:5023]: +1

**Review by XDelta — APPROVED** [review]
Non-blocking on both:
AssemblyCopyright set to 2022

**Pull request merged**